lib/rest_in_peace/response_converter.rb

```diff
@@ -42,7 +42,14 @@ def convert_from_array
     end
 
     def convert_from_hash(entity = body, instance = existing_instance)
-      instance.force_attributes_from_hash entity
+      entity_with_indifferent_access = entity.with_indifferent_access
+
+      # We received a validation error and we need to add it to the model.
+      if entity_with_indifferent_access[:status] == 422
+        instance.errors.add(:base, entity_with_indifferent_access[:message])
```

Comment:
Good point! And that's actually a question I still have. Do you see any use cases where we would need an active model instance but not the middleware? Because it seems to me that if you let the `422` errors pass with the raise errors middleware, you'll most likely have a rails instance in the background...What do you think?